src/status_im2/navigation/events.cljs

```diff
@@ -143,8 +143,14 @@
 (rf/defn dismiss-all-overlays
   {:events [:dissmiss-all-overlays]}
   [{:keys [db]}]
-  {:dissmiss-all-overlays-fx nil
-   :db                       (-> db
-                                 (dissoc :popover/popover)
-                                 (dissoc :visibility-status-popover/popover)
-                                 (assoc-in [:bottom-sheet :hide?] true))})
+  {:dispatch-n [[:hide-popover]
+                [:hide-visibility-status-popover]
+                [:hide-bottom-sheet]
+                [:bottom-sheet-hidden]
```

Comment:
For dismissing bottom sheet, we also need to dispatch `:bottom-sheet-hidden` after `hide-bottom-sheet` this was causing the original issue https://github.com/status-im/status-mobile/issues/15771

Reply:
Its weird we have to dispatch two separate events